sdk/python/tests/compiler/testdata/any_sequencer.yaml

```diff
@@ -123,12 +148,14 @@ spec:
         - name: pipelineRun-namespace
         steps:
         - args:
-          - -namespace
+          - --namespace
           - $(params.pipelineRun-namespace)
-          - -prName
+          - --prName
           - $(params.pipelineRun-name)
-          - -taskList
+          - --taskList
           - task1,task2,task3
+          - -c
```

Comment:
@pugangxa looks like it needs double dashes flag `--c`. Right now I'm seeing this error
```
flag provided but not defined: -c
Usage of any-taskrun:
  -namespace string
    	The namespace of the pipelinerun.
  -prName string
    	The name of the pipelinerun.
  -taskList string
    	The comma separated list of the tasks.
```

Reply:
or maybe you haven't update the image?

Reply:
Yes Tommy I haven't update the image, I should update it after code merged right?